app/styles/base/_config.scss

```diff
@@ -72,6 +72,7 @@ $easing4: cubic-bezier(0.25, 1.4, 0.18, 1); // try with 200ms
 $easing6: cubic-bezier(0.56, 1, 0.86, 1.139);
 
 // Layout: Breakpoints
+$layout-xs: 333px;
```

Comment:
is this different from below 480px? wondering if we need to have both of them

Reply:
yes, because it works also for mediaqueries such as min-height (and 480 is too big already)